infra/utils/merge_dependabot_prs/merge_dependabot_prs.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Prerequisites:
+#   Install and Authenticate Github CLI (https://github.com/cli/cli)
+#
+# Usage of merge_dependabot_prs.sh:
+#   merge_dependabot_prs.sh [GitHub Org]
+#
+#   [GitHub Org] defaults to terraform-google-modules
+
+set -u
+
+# Variables - Configurable
+ORG=${1:-terraform-google-modules}
+FLABEL="dependabot-checks-failed"
+
+# Initialize Variables
+FPRS=()
+PPRS=()
+
+# Check gh is installed
+if [ ! -x $(which gh) ]; then
+  echo "GitHub CLI (gh) not found - Install from https://github.com/cli/cli"
+  exit 1
+fi
+
+# Check gh is authenticated
+gh auth status > /dev/null 2>&1
+if [ ! $? -eq 0 ]; then
+  echo "Please authenticate GitHub CLI with 'gh auth login' prior to running"
+  exit 1
+fi
+
+# Retrieve list of repos in the Org
+REPOS=`gh repo list $ORG --no-archived --json name -q .[].name -L 1000`
+
+# Confirm we retrieved at least 1 repo
+if [ "${REPOS[@]}" == 0 ]; then
+  echo "No repos found for $ORG"
+  exit
+fi
+
+# Process the repos
+for REPO in $REPOS; do
+  REPO=$ORG/$REPO
+  echo "Processing $REPO"
+
+  # Retrieve Pull Requests
+  PRS=`gh pr list -R $REPO -s open --json number -q '.[].number' --app dependabot`
+  if [ "${PRS[@]}" > 0 ]; then
+    echo "  Processing ${PRS[@]} Pull Requests"
+  else
+    echo "  No open Dependabot Pull Requests found for $REPO"
+  fi
+
+  # Process Pull Rquests
+  for PR in $PRS; do
+    echo $PR
+    # Check status of Pull Request Checks
+    gh pr checks $PR -R $REPO > /dev/null 2>&1
+    if [ $? -eq 0 ]; then
+      PPRS+=($PR)
+      # Approve the Pull Request
+      gh pr review $PR -R $REPO --approve -b "Author: Dependabot<br />Checks: Passing"
```

Comment:
I think LGTMing is enough? 
```suggestion
      gh pr review $PR -R $REPO --approve -b "LGTM"
```